app/helpers/drop.js

```diff
@@ -0,0 +1 @@
+export { default, drop } from 'ember-functional-helpers/helpers/drop';
```

Comment:
Can we also export in https://github.com/DockYard/ember-functional-helpers/blob/remove-compose/addon/index.js please